src/commands/user/export-mapping.ts

```diff
@@ -0,0 +1,42 @@
+import fs from 'fs'
+import path from 'path'
+
+import { flags } from '@oclif/command'
+import { Kommand } from '../../common'
+import { kuzzleFlags } from '../../support/kuzzle'
+
+export default class UserExportMapping extends Kommand {
+  static description = 'Exports user mapping to JSON.'
+
+  static flags = {
+    help: flags.help({}),
+    path: flags.string({
+      description: 'Dump directory',
+      default: 'userMapping'
+    }),
+    ...kuzzleFlags,
+    protocol: flags.string({
+      description: 'Kuzzle protocol (http or websocket)',
+      default: 'ws'
+    })
+  }
+
+  async runSafe() {
+    const filename = path.join(this.flags.path, 'userMapping.json')
+
+    this.logInfo(`Exporting user mapping in ${filename} ...`)
+
+    fs.mkdirSync(this.flags.path, { recursive: true })
+
+    const mapping = await this.sdk?.security.getUserMapping()
+
+    const dump = {
+      type: 'userMapping',
```

Comment:
About the kebab-case notation, IMHO it's more readable because it's closer to the natural sentence construction (word separated by spaces). Since variable names cannot contains `-`, we use camelCase (or snake_case which I prefer for the same reason)
```suggestion
      type: 'users-mappings',
```